@xen-orchestra/web-core/lib/components/ui/info/UiInfo.vue

```diff
@@ -42,7 +42,6 @@ const icon = computed(() => iconByAccent[props.accent])
 
 <style lang="postcss" scoped>
 .ui-info {
-  align-items: start;
```

Comment:
Be careful when updating the styles of core UI components. This property is useful here to keep the icon properly aligned when the text in the `UiInfo` is more than one line long ([example here](https://www.figma.com/design/l2O2VvzJRnOCvqxhM7d124/1.-Vates---Product-Design-System?node-id=1456-31144&t=zqyX6g7OWypM3k2X-4)).
However, I agree that in your case, the icon alignment could be better. You can add a class in `PoolNetworksPifStatus` and use it to change the alignment, see my comment on the component.

Reply:
I agree to be careful when updating core UI components. 
But shouldn't we remove `align-items: start`  and use if we need it? I have a feeling we'll need `align-items: center` more often. no ?

Reply:
Yes, maybe that would be better to handle single line info by default and add an override if necessary. Or we can add a prop for that? @ByScripts